shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/server/ActionProcessor.java

```diff
@@ -1663,6 +1662,11 @@ private void update() {
                         && !entry.getValue().updateInProgress
                         && Duration.between(entry.getValue().lastChecked, now).toMinutes()
                             >= Math.max(10, entry.getKey().retryMinutes()))
+            /**
+             * Sort by time since last checked, and then priority, to avoid starving actions of
+             * attention because of priority then sort by their ActionState's processPriority so
+             * that certain ActionStates get checked first.
+             */
```

Comment:
"to avoid starving actions of attention because of priority" I'm unclear on which clause this belongs with - does it belong with "Sort by time since last checked, and then priority" or "then sort by their ActionState's processPriority..."?

Reply:
Added a comma.